### PR TITLE
[PL-43530] [CDS-86689] Delegate 23.12.81804 hotfix release notes

### PR DESCRIPTION
**Pull request by brian-f-harness**
Delegate 23.12.81804 hotfix release notes

For reviewers, a preview is available.

https://657a26a8b802a514f80ec781--harness-developer.netlify.app/release-notes/delegate#version-231281804

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/delegate.md [hunk 1]
```diff
@@ -77,6 +77,16 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
 
    This issue has been resolved, and the UI now displays "User added successfully".
 
+### Version 23.12.81604
```
Comment (brian-f-harness) [inline release-notes/delegate.md:80]:
```suggestion
### Version 23.12.81804
```

File: release-notes/delegate.md [hunk 2]
```diff
@@ -77,6 +77,16 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
 
    This issue has been resolved, and the UI now displays "User added successfully".
 
+### Version 23.12.81804
+
+#### Hotfix
+
+- You can now use a Refresh token in the Tanzu connector for all API communications. If a Refresh token isn't provided, Harness will use the username and password for the API calls. (CDS-86689)
+
+   :::info note
```
Comment (brian-f-harness) [inline release-notes/delegate.md:86]:
```suggestion
:::info note
```

File: release-notes/delegate.md [hunk 3]
```diff
@@ -77,6 +77,16 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+Currently, this feature is behind the feature flag `CDS_CF_TOKEN_AUTH`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+:::
```
Comment (brian-f-harness) [inline release-notes/delegate.md:88]:
```suggestion

:::
```

**Review by thisrohangupta — APPROVED** [review]
this looks good

**Pull request merged**